runtime/v2/example/example.go

```diff
@@ -20,6 +18,7 @@ package example
 
 import (
 	"context"
+	"github.com/containerd/ttrpc"
```

Comment:
sort imports
```go
import (
	"context"
	"os"

	taskAPI "github.com/containerd/containerd/api/runtime/task/v2"
	"github.com/containerd/containerd/errdefs"
	"github.com/containerd/containerd/pkg/shutdown"
	"github.com/containerd/containerd/plugin"
	ptypes "github.com/containerd/containerd/protobuf/types"
	"github.com/containerd/containerd/runtime/v2/shim"
	"github.com/containerd/ttrpc"
)
```

Reply:
Oh, CI was trying to tell me that my imports are not formatted properly, understood. I fixed it manually. For the future: can `goimports` format them automatically?

Reply:
Usually the ide or vim plugin can goimports automatically, and the ci will only check the code, not modify it.